build/release/teamcity-mark-build.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+source "$(dirname "${0}")/teamcity-support.sh"
+
+tc_start_block "Variable Setup"
+build_label=$1
+build/builder.sh make .buildinfo/tag
+build_name="${TAG_NAME:-$(cat .buildinfo/tag)}"
+release_branch="$(echo "$build_name" | grep -Eo "^v[0-9]+\.[0-9]+")"
+
+if [[ -z "${DRY_RUN}" ]] ; then
+  google_credentials=$GOOGLE_COCKROACH_CLOUD_IMAGES_CREDENTIALS
+  gcr_repository="us.gcr.io/cockroach-cloud-images/cockroach"
+else
+  # TODO: Use dry run locations other than jamesl*
+  google_credentials=$GOOGLE_JAMESL_TEST_CREDENTIALS
+  gcr_repository="gcr.io/jamesl-test/cockroach-pre-release"
```

Comment:
Here are the long-term testing locations.

```suggestion
  google_credentials=$GOOGLE_COCKROACH_RELEASE_CREDENTIALS
  gcr_repository="us.gcr.io/cockroach-release/cockroach-test" 
```